lectures/Haskell/src/Typeclasses.hs

```diff
@@ -39,8 +39,8 @@ Haskell implements overloading, which is the ability to define an operation
 for more than one types, using type classes.
 
 The Num type class defines operations like +, -, * for numeric types. Every
-type that implements the type class, i.e., provides an instance of the type
-class, should define this operations.
+type that implements the type class, i.e. Int, provides an instance of the type
```

Comment:
```suggestion
type that implements the type class, i.e. provides an instance of the type
```

the original meaning is the intended one. Could be parenthesised for more clarity.